rpcs3/Emu/RSX/Overlays/overlay_trophy_notification.cpp

```diff
@@ -34,17 +34,20 @@ namespace rsx
 
 		trophy_notification::trophy_notification()
 		{
-			frame.set_pos(0, 0);
-			frame.set_size(300, 80);
-			frame.back_color.a = 0.85f;
-
-			image.set_pos(8, 8);
-			image.set_size(64, 64);
+			frame.set_pos(68.24, 55.34);
```

Comment:
the parameters you pass are doubles, but set_pos only accepts u16 (a.k.a. no decimal places)